### PR TITLE
fix(integrations): Fix dropdowns for project mapping

### PR DESCRIPTION
**Pull request by roggenkemper**
this pr fixes the dropdowns for project mapping. 

before:
Screenshot 2023-11-15 at 2 03 04 PM

after: 
Screenshot 2023-11-15 at 2 03 30 PM

### CODE REVIEW

File: static/app/components/forms/fields/projectMapperField.tsx [hunk 1]
```diff
@@ -173,80 +194,25 @@ export class RenderField extends Component<RenderProps, State> {
...
-              ValueContainer: customMappedValueContainer,
+              SingleValue: containerProps => {
+                return (
+                  <components.ValueContainer {...containerProps}>
+                    <MappedValueContainer>
+                      {getIcon(iconType)}
+                      {mappedItemsByValue[selectedMappedValue || ''].label}
+                    </MappedValueContainer>
+                  </components.ValueContainer>
+                );
```
Comment (scttcper) [inline static/app/components/forms/fields/projectMapperField.tsx:214]: i don't think we need these anymore?

Comment (scttcper) [reply static/app/components/forms/fields/projectMapperField.tsx:214]: oh it kinda lines stuff up. hmmm

Comment (scttcper) [reply static/app/components/forms/fields/projectMapperField.tsx:214]: what if we did something more like this and got the props out of the containerProps
```suggestion
              SingleValue: containerProps => {
                return (
                  <components.ValueContainer {...containerProps}>
                    <MappedValueContainer>
                      {containerProps.data.leadingItems}
                      {containerProps.children}
                    </MappedValueContainer>
                  </components.ValueContainer>
                );
```

**Review by scttcper — APPROVED** [review]
looks good, my feedback is nit picking

**Review by scefali — APPROVED** [review]

**Pull request merged**